examples/cmake-examples/QmlBarcodeReader/cmake/Locations.cmake

```diff
@@ -0,0 +1,39 @@
+set(COMPANY "Scythe Studio")
+set(COPYRIGHT "Copyright (c) 2022 Scythe Studio. All rights reserved.")
+set(IDENTIFIER "com.scythestudio.scodes.example")
+
+# ---CONFIGURATION---
+# MOST IMPORTANT PART -> change to ON/OFF if you want to use these elements in your project
+option(USE_QML "Add QML support" ON)
+option(USE_LIBS "Use external libraries" ON)
+
+set(QT_NAME Qt5)
+
+# Locations - directories in project structure
+set(LIB_DIR "../../../src")
```

Comment:
Such type of path should be avoided - you should rather start with variables like ${CMAKE_SOURCE_DIR}